demisto_sdk/commands/format/format_module.py

```diff
@@ -114,7 +114,7 @@ def format_manager(input: str = None,
                 current_excluded_files.remove('CommonServerPython.py')
             if os.path.basename(file_path) in current_excluded_files:
                 continue
-            if dirname.endswith('test_data') or dirname.endswith('doc_imgs'):
+            if dirname.lower().endswith('test_data') or dirname.lower().endswith('testdata') or dirname.lower().endswith('doc_imgs'):
```

Comment:
we actually have that in validate:
`any(test_dir in str(file_path) for test_dir in TESTS_AND_DOC_DIRECTORIES):`
better use this here as well.

Reply:
Fixed by https://github.com/demisto/demisto-sdk/pull/1465/commits/d832924233566e6e4b937a7d8f30e9956eca4374